core-bundle/src/Resources/contao/templates/_new/content_element/_base.html.twig

```diff
@@ -0,0 +1,26 @@
+{% if not searchable|default(true) %}<!-- indexer::stop -->{% endif %}
```

Comment:
see ce1fd482603824960193bb2a798e6c5a3aed2c99

Should be identical to your suggestion but has the end tags in a new line plus Twig whitespace control, so that it is not output.